packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/update_objects_spaces.ts

```diff
@@ -19,22 +19,18 @@ import type {
   SavedObjectsUpdateObjectsSpacesResponseObject,
 } from '@kbn/core-saved-objects-api-server';
 import {
-  AuditAction,
+  SavedObject,
   type ISavedObjectsSecurityExtension,
   type ISavedObjectTypeRegistry,
   type SavedObjectsRawDocSource,
 } from '@kbn/core-saved-objects-server';
-import {
-  SavedObjectsErrorHelpers,
-  ALL_NAMESPACES_STRING,
-  type DecoratedError,
-  SavedObjectsUtils,
-} from '@kbn/core-saved-objects-utils-server';
+import { ALL_NAMESPACES_STRING } from '@kbn/core-saved-objects-utils-server';
+import { SavedObjectsErrorHelpers, type DecoratedError } from '@kbn/core-saved-objects-common';
 import type {
   IndexMapping,
   SavedObjectsSerializer,
 } from '@kbn/core-saved-objects-base-server-internal';
-import type { SavedObject } from '@kbn/core-saved-objects-server';
+import { AuthorizeObjectWithExistingSpaces } from '@kbn/core-saved-objects-server/src/extensions/security';
```

Comment:
NIT: unnecessary nested import. Please import from `@kbn/core-saved-objects-server` (and export accordingly if required)